docs/TerraformPlanResultsPR.md

```diff
@@ -0,0 +1,47 @@
+## [TerraformPlanResultsPR](../src/TerraformPlanResultsPR.groovy)
+
+Use this to post Terraform plan results in the comments of a PR.
+
+One-Time Setup:
+* Install the terraform-landscape gem on your Jenkins slaves. (ONLY when using `withLandscape(true)`)
+
+Requirements:
+* Enable CredentialsPlugin to set the GITHUB_TOKEN environment varaible. This is later used in the TerraformPlanResultsPR Plugin for authentication.
+* Enable AnsiColorPlugin for colors in Jenkins (ONLY when usng `withLandscape(true)`)
+
+Configuration Methods:
+* `withRepoHost(String)`: specify the Github source address (DEFAULT: "https://api.github.com/")
+* `withRepoSlug(String)`: specify the full repo slug (DEFAULT: "")
+* `withLandscape(bool)`: enable or disable the terraform landscape gem for plan output (DEFAULT: false)
+* `withGithubTokenEnvVar(String)`: specify the environment variable used for github auth (DEFAULT: "GITHUB_TOKEN")
+
+Important Notes:
+* This plugin supports terraform plan output with the terraform landscape gem, but there is currently a standalone plugin `TerraformLandscapePlugin` for this feature.
+* Note the `TerraformLandscapePlugin` does NOT contain support to post comments on pull requests.
+* If you desire BOTH terraform landscape output and comments on PR's, you should use the `TerraformPlanResultsPR` plugin with `withLandscape(true)`
+
+```
+@Library(['terraform-pipeline']) _
+
+Jenkinsfile.init(this)
+
+// Required to set GITHUB_TOKEN envionrment variable for use with TerraformPlanResultsPR Plugin
+// FOO/GITHUB_TOKEN will contain the respective username/password values of the 'my-cred' credential.
+CredentialsPlugin.withBuildCredentials([usernameVariable: 'FOO', passwordVariable: 'GITHUB_TOKEN'], 'my-cred').init()
+
+AnsiColorPlugin.init()                                           // Required when using 'withLandscape(true)'
```

Comment:
What does 'Required  when using `withLandscape` mean?  What happens if I enable landscape and do not enable  the AnsiColorPlugin?

Reply:
I have updated the docs to expand on this. When using the Landscape gem,AnsiColor is required for Jenkins to colorize the output.

Reply:
If you enable landscape without AnsiColor, you will see the plan results from `terraform plan | landscape` without color